azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachines.java

```diff
@@ -341,4 +341,55 @@
      * @return handle to the asynchronous execution
      */
     Observable<RunCommandResult> runCommandAsync(String groupName, String name, RunCommandInput inputCommand);
+
+    /**
+     * Force delete a resource from Azure, identifying it by its resource ID.
+     *
+     * @param id the resource ID of the resource to delete
+     */
+    void forceDeleteById(String id);
```

Comment:
Personally I prefer overload of `deleteById(id, force)` a little bit.

You can keep this unless anyone else got preference as well.